content/information-distribution/node.md

```diff
@@ -82,58 +82,20 @@ provide for this room.
 
 ### Reaching an entry node
 
-Because the list of aliases only list the nodes' server names, and not their
-FQDN, a node might not be reachable at the address defined by its server name.
+Because processing the list of aliases only list the nodes' server names, and
```

Comment:
this sentence does not seems correct to me
```suggestion
Because the list of aliases references only the nodes' server names, and
```
```suggestion
Because processing the list of aliases gives only the nodes' server names, and
```
or 
`Because the list of room aliases uses the server names and not their FQDN, the user may not be able to reach the server by simply resolving its name. As an example, ...`

or something else?